app/patients/communication/controller.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```

Comment:
I think a better practice would be not to mix existing and new (not finished) features. Please, split your PR.